### DISCUSSION TITLE
Unable to get a custom route to update a user profile to work

### DISCUSSION THREAD

**Discussion by michael-lawrence** [strapi/strapi#7429] [Legacy Topics]
Hi, 
I've followed the instructions located at https://github.com/strapi/strapi/issues/2818 and I'm unable to get the custom PUT /users/me route to work. I get 403 errors when trying to PUT at /users/me. I've made sure to give access to this route in the admin for both authenticated and public users (just in case). I'm passing the jwt token the same way I do when calling the GET /users/me route. I even added a console.log statement inside of the User.updateme method to see if it's even getting called and I never see my logged message in the console. Any idea why this isn't working? Am I missing something?

<img width="671" alt="Screen Shot 2020-08-11 at 1 01 28 PM" src="https://user-images.githubusercontent.com/569562/89926523-0c599000-dbd3-11ea-8e45-eac177954380.png">

<img width="1006" alt="Screen Shot 2020-08-11 at 1 01 52 PM" src="https://user-images.githubusercontent.com/569562/89926543-111e4400-dbd3-11ea-98a5-417c4c22582e.png">

<img width="932" alt="Screen Shot 2020-08-11 at 1 02 07 PM" src="https://user-images.githubusercontent.com/569562/89926562-167b8e80-dbd3-11ea-9281-b639016a8b5b.png">

```json
{
    "statusCode": 403,
    "error": "Forbidden",
    "message": "Forbidden"
}
```

**Comment by derrickmehaffy** [comment]
Hi @michael-lawrence I did a quick test just to see if I could get it to console.log and it appears to be working fine:
![image](https://user-images.githubusercontent.com/8593673/90191546-f2a47e00-dd75-11ea-94cd-4ff81e491d0e.png)
![image](https://user-images.githubusercontent.com/8593673/90191557-f801c880-dd75-11ea-922a-72f107ddf277.png)

You did set the permissions correct?

![image](https://user-images.githubusercontent.com/8593673/90191605-0cde5c00-dd76-11ea-91bd-e85587c1a478.png)

**Reply by michael-lawrence** [reply]
Actually, it wasn't permissions as I had set that up already. I just noticed in your PostMan body you used raw json and I was using form-data. Thanks so much for your help!
